lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2735,19 +2735,14 @@ def calc_arrow(uvw, angle=15):
 
     def voxels(self, *args, facecolors=None, edgecolors=None, **kwargs):
         """
-        ax.voxels([x, y, z,] /, filled, **kwargs)
+        ax.voxels([x, y, z,] /, filled, facecolors=fc, edgecolors=ec, **kwargs)
```

Comment:
Should the kwargs be `None` instead?

Reply:
sorry, had just copy-pasted from below, now fixed.